man/systemd-sysext.xml

```diff
@@ -107,13 +106,14 @@
     <para>Note that there is not concept of enabling/disabling installed system extension images: all
     installed extension images are automatically activated at boot.</para>
 
-    <para>A simple mechanism for version compatibility is enforced: a system extension image may carry an
-    <filename>/etc/os-release</filename> file that is compared with the host <filename>os-release</filename>
-    file: the contained <varname>ID=</varname> fields have to match, as well as the
-    <varname>SYSEXT_LEVEL=</varname> field (if defined). If the latter is not defined the
-    <varname>VERSION_ID=</varname> field has to match instead. System extensions should not ship a
-    <filename>/usr/lib/os-release</filename> file (as that would be merged into the host
-    <filename>/usr/</filename> tree, overriding the host OS version data, which is not desirable).</para>
+    <para>A simple mechanism for version compatibility is enforced: a system extension image must carry a
+    <filename>/usr/lib/extension-release.d/extension-release.$name</filename> file, which must match its
+    image name, that is compared with the host <filename>os-release</filename> file: the contained
+    <varname>ID=</varname> fields have to match, as well as the <varname>SYSEXT_LEVEL=</varname> field (if
+    defined). If the latter is not defined, the <varname>VERSION_ID=</varname> field has to match instead.
+    System extensions should not ship a <filename>/usr/lib/os-release</filename> file (as that would be
+    merged into the host <filename>/usr/</filename> tree, overriding the host OS version data, which is not
+    desirable).</para>
```

Comment:
maybe say explicitly that the extension-release files follow the same format and semantics and carry the same contents as the os-release files of the OS itself, except they describe the OS rources in the image.

Reply:
done